CONTRIBUTING.md

```diff
@@ -35,20 +35,20 @@ To make a suggestion [file an issue](https://github.com/google/open-location-cod
 
 If you are intending to implement, please see the [Contributing code](#contributing-code) section below for next steps.
 
-If you are adding Open Location Codes to your project, please contact the [Open Location Code Google Group](https://groups.google.com/forum/#!forum/open-location-code) so we can suggest how you can make the most of the codes.
+If you are adding Open Location Code to your project, please contact the [Open Location Code Google Group](https://groups.google.com/forum/#!forum/open-location-code) so we can suggest how you can make the most of the codes.
 
 ## Contributing code
 
 The Open Location Code project accepts and greatly appreciates code contributions!
 
 A few things to note:
 
-* The Open Location Code project follows the [fork & pull](https://help.github.com/articles/using-pull-requests/#fork--pull) model for accepting contributions.
-* We follow [Google's JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html).
+- The Open Location Code project follows the [fork & pull](https://help.github.com/articles/using-pull-requests/#fork--pull) model for accepting contributions.
```

Comment:
*, - and + are all valid list markers, see https://daringfireball.net/projects/markdown/syntax#list. Let's leave them be.